build.yaml

```diff
@@ -154,6 +154,10 @@ steps:
    image:
      valueFrom: create_certs_image.image
    script: |
+     until ! kubectl get secret -n {{ default_ns.name }} ssl-config-hail-root 2>/dev/null
```

Comment:
Could you add a comment explaining why we're doing this?
(The dev certificates expires after a month.)

Reply:
Done!